coalib/bears/requirements/PythonRequirement.py

```diff
@@ -5,6 +5,37 @@ class PythonRequirement(PackageRequirement):
     """
     This class is a subclass of ``PackageRequirement``, and helps specifying
     requirements from ``pip``, without using the manager name.
+
```

Comment:
this was added here as there was no way to test it in the base class, and this solves coverage.